kod/util/rntmaint.kod

```diff
@@ -391,7 +445,34 @@ messages:
 
       % RenewRental returns the new days left.
       return send(oRoom,@RenewRental,#iTimeAdded=iDays);
-   }      
+   }
+
+   FindKeyByPlayer(who=$, iLocation=$)
+   "Returns the gold room key matching the specified inn RID held by the "
```

Comment:
The key it returns is always gold?  If that ever changes color, someone will know to change this comment?  You get the point....

Reply:
No, that's just the name of the key.  But I see exactly what you mean and have changed it to reference it as its class name (RoomKey) as well as what its purpose is (master/original key).